liquibase-core/src/main/java/liquibase/command/core/InternalSnapshotCommandStep.java

```diff
@@ -112,19 +112,15 @@ public static String printSnapshot(CommandScope commandScope, CommandResults sna
 
         return SnapshotSerializerFactory.getInstance().getSerializer(format.toLowerCase(Locale.US)).serialize((DatabaseSnapshot) snapshotResults.getResult("snapshot"), true);
     }
-//
-//        public void merge(SnapshotCommandResult resultToMerge) {
-//            this.snapshot.merge(resultToMerge.snapshot);
-//        }
-//    }
 
     public static void logUnsupportedDatabase(Database database, Class callingClass) {
         if (LicenseServiceUtils.isProLicenseValid()) {
             if (!(database instanceof MSSQLDatabase
                     || database instanceof OracleDatabase
                     || database instanceof MySQLDatabase
                     || database instanceof DB2Database
-                    || database instanceof PostgresDatabase)) {
+                    || database instanceof PostgresDatabase
+                    || database.getShortName().equalsIgnoreCase("snowflake"))) {
```

Comment:
Do we have to do this hacky because adding a dependency on the snowflake module would cause a circular dependency?

Reply:
Yep. Until that module is fully integrated into core we've got to use this hack. I talked it through with @nvoxland but I should probably also add a TODO. I'll do that now!

Reply:
🤮